### PR TITLE
Feat/cat redux

### PR DESCRIPTION
**Pull request by qrispi**
Change Summary:
- This merge adds a catSlice to the store with reducer actions to assign and store a name for the cat, & store and increment/decrement the cat's morality.
- Choice buttons are also connected to the reducer actions so that clicking a "good" choice increments the cat's morality by 1 and clicking a "bad"choice decrements the cat's morality by 1.

Reducer Actions: 
- updateCatName: Stores the cat's name as a string once the user types and submits the text input box
- incrementCatMorality: Increases cat's morality by 1 upon clicking a "good" choice
- decrementCatMorality: decreases cat's morality by 1 upon clicking a "bad" choice

### CODE REVIEW

**Review by BriBourassa — APPROVED** [review]
Everything's progressing smoothly :)

**Review by BriBourassa — APPROVED** [review]

File: src/components/Choices/Choices.js
Comment (Jesusborjas006) [inline src/components/Choices/Choices.js:1]: Great job using dispatch to update the state of the cat's morality.

**Review by jsahim — APPROVED** [review]
Making great progress! 😺

File: src/features/catSlice.js
Comment (Jesusborjas006) [inline src/features/catSlice.js:1]: Naming of these reducers are easy to follow!

**Review by Jesusborjas006 — APPROVED** [review]

**Pull request merged**